tools/test.py

```diff
@@ -1253,6 +1268,9 @@ def BuildOptions():
       default=False, action="store_true")
   result.add_option("--cat", help="Print the source of the tests",
       default=False, action="store_true")
+  result.add_option("--flaky-tests",
+      help="Regard tests marked as flaky (run|skip|dontcare)",
```

Comment:
I guess "ignore" would have worked as well as "dontcare". Since this is already used in v0.x, and CI rules, I'd rather leave it unchanged